src/main/java/nl/tudelft/goalkeeper/parser/results/files/module/ModuleFile.java

```diff
@@ -12,10 +18,15 @@
 /**
  * Class which holds data on a module file.
  */
-public class ModuleFile extends File implements Module {
+public class ModuleFile extends File implements Module, Linguistic {
```

Comment:
Will all the expressions also be added to this object?

Reply:
What do mean you mean exactly?

Reply:
Well we have at the highest level currently `Rules` which contains conditions etc but these `Rule` need to be associated with a module file. (in a set or something). Is it an idea to that here?